ui/lib/core/addon/components/empty-state.hbs

```diff
@@ -8,19 +8,19 @@
     {{#if @icon}}
       <div class="empty-state-icon">
         <Icon @name={{@icon}} @size="24" />
-        <h3 class="empty-state-title" data-test-empty-state-title>
+        <p class="empty-state-title" data-test-empty-state-title>
```

Comment:
Same here for the HDS text component if we want to use!

Reply:
This one I'm going to leave as is, since we should just replace with Hds::ApplicationState in the future